tests/test_ates.py

```diff
@@ -11,9 +11,18 @@
 class TestAtes(TestCase):
     def test_ates(self):
         """
-        This test checks the constraints concerning the heat2discharge and energy conservation for
-        the ates. THe heat loss model used annd the typical cyclic constraint that will be applied
-        in most use cases.
+        This test checks the constraints concerning the heat to discharge and energy conservation
```

Comment:
"Test the constraints ... " or "Check the constraints ..."

Reply:
done